test/Cronofy.Test/CronofyAccountClientTests/GetProfiles.cs

```diff
@@ -30,6 +31,13 @@ public void CanGetCalendars()
       ""profile_name"": ""example@cronofy.com"",
       ""profile_connected"": false,
       ""profile_relink_url"": ""http://to.cronofy.com/RaNggYu""
+    },
```

Comment:
In the above block, the `apple` `provider_name` would be `icloud` to best match a real response